packages/react-conformance-make-styles/src/matchers/toHaveMergeClassesCalledTimesWithClassName.ts

```diff
@@ -0,0 +1,46 @@
+import { mergeClasses } from '@fluentui/react-make-styles';
+
+declare global {
+  namespace jest {
+    interface Matchers<R> {
+      toHaveMergeClassesCalledTimesWithClassName(className: string, times: number): R;
+    }
+  }
+}
+
+type MergeClassesParams = Parameters<typeof mergeClasses>;
+
+/**
+ * Custom Jest matcher that implements a check to ensure that mergeClasses() function with a passes class got called
+ * exact number of times.
+ */
+export const toHaveMergeClassesCalledTimesWithClassName: jest.CustomMatcher = (
+  result: jest.Mock<{}, MergeClassesParams>['mock']['calls'],
+  className: string,
+  expectedTimes: number,
+) => {
+  let callCount: number = 0;
+
+  let pass = true;
+  let message = '';
+
+  result.forEach(classesList => {
+    const indexInClassList = classesList.indexOf(className);
```

Comment:
you could just use `includes` or `find` I don't think we need IE11 comapt in jest ;)

Reply:
Yeah, it's required in an another matcher as it uses that index later to compare that `className` will be last one, but there `includes()` is good enough 👍